Resources/public/js/views/services/ez-contentcreateviewservice.js

```diff
@@ -93,6 +97,50 @@ YUI.add('ez-contentcreateviewservice', function (Y) {
                 });
             });
         },
+
+        /**
+         * changeLanguage event handler. It opens languageSelectionBox for selecting
+         * language of created content
+         *
+         * @method _changeLanguage
+         * @private
+         * @param {EventFacade} e
+         */
+        _changeLanguage: function (e) {
+            var that = this;
+
+            e.preventDefault();
+            this.fire('languageSelect', {
+                config: {
+                    title: "Change language to:",
+                    languageSelectedHandler: Y.bind(this._changeContentLanguage, this, e.target),
```

Comment:
same remark here, the method name is bad because I have to look at the code to have rough idea of it does